core/app/models/spree/property.rb

```diff
@@ -7,6 +7,7 @@ class Property < Spree::Base
     has_many :products, through: :product_properties
 
     validates :name, :presentation, presence: true
+    validates :name, uniqueness: true
```

Comment:
@vinay-mittal Please add an option 'allow_blank: true' to skip this validation in case presence fails.